sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_models.py

```diff
@@ -553,29 +551,31 @@ def _from_generated(cls, cell, page, read_result):
 
 
 class CustomFormModel(object):
-    """Represents a custom model returned from training.
+    """Represents a model trained from custom forms.
```

Comment:
trained on custom forms?

Reply:
"On" doesn't sound right to me. Anne suggested this change for the docstring so i'm going to keep it.